### ISSUE TITLE
Bubble mode shows an unnecessary horizontal scrollbar.

### ISSUE THREAD

**Issue by Pauan** [Pauan/tab-organizer#19]
Originally reported on Google Code with ID 19

```
What steps will reproduce the problem?
1. Turn on "Bubble mode" in the options.
2. Make sure the popup's width is greater than 800px.
3. Open Tab Organizer.

What is the expected output? What do you see instead?
When there is only 1-4 windows visible, there should not be a horizontal scrollbar.
Instead, there is *always* a scrollbar, even when it is not needed.

Please provide any additional information below.
The technical reason for this, is that the bubble has a max width of 800px. Because
the popup's contents are not in the flow, I need to use a bit of a hack to make bubble
mode work. Unfortunately, this hack causes horizontal scrollbars.

I can't use `overflow: hidden` because we still need to show the scrollbar when the
user has 5+ windows visible. The simple fix for users is to set the popup's width to
800px.

It appears that setting `max-width: 100%` on the <body> element fixes this.
```

Reported by `pcxunlimited` on 2010-09-05 17:09:22

**Comment by Pauan** [comment]
```
This was fixed with revision 2b8eee1ec0 and should be in version 2.6.
```

Reported by `pcxunlimited` on 2010-09-05 17:25:04

**Comment by Pauan** [comment]
```
Released with version 2.6.
```

Reported by `pcxunlimited` on 2010-09-06 01:20:26
- **Status changed**: `Verified`

**Comment by Pauan** [comment]
Reported by `pcxunlimited` on 2012-08-27 23:44:47
- **Status changed**: `Fixed`
